test-utils/src/main/java/org/sonarsource/sonarlint/core/test/utils/SonarLintBackendFixture.java

```diff
@@ -406,20 +396,20 @@ public SonarLintBackendBuilder withDisabledPluginsForAnalysis(String pluginKey)
     }
 
     public SonarLintBackendBuilder withSecurityHotspotsEnabled() {
-      this.areSecurityHotspotsEnabled = true;
+      this.backendCapabilities.add(BackendCapability.SECURITY_HOTSPOTS);
       return this;
```

Comment:
At some point it might be better to replace all those methods with just `withBackendCapability(BackendCapability)`. But this can come later